src/Config/Registrar.php

```diff
@@ -18,10 +20,12 @@ public static function Filters(): array
     {
         return [
             'aliases' => [
-                'session'    => SessionAuth::class,
-                'tokens'     => TokenAuth::class,
-                'chain'      => ChainAuth::class,
-                'auth-rates' => AuthRates::class,
+                'session'           => SessionAuth::class,
+                'tokens'            => TokenAuth::class,
+                'chain'             => ChainAuth::class,
+                'auth-rates'        => AuthRates::class,
+                'group-filter'      => GroupFilter::class,
+                'permission-filter' => PermissionFilter::class,
```

Comment:
Wouldn't the key names be `group` and `permission`?

Reply:
https://github.com/codeigniter4/shield/pull/270#issuecomment-1214029245

Reply:
@jlopes90 `group  `and `permission `is Okay.